community/rspamd/APKBUILD

```diff
@@ -69,9 +66,11 @@ build() {
 	make
 }
 
-package() {
-	cd "$builddir"
+check() {
+	make test
+}
```

Comment:
`make test` does nothing. The make command is: `make rspamd-test` - https://github.com/rspamd/rspamd/blob/master/.circleci/config.yml#L67-L93 & luarocks would be needed as a checkdepends.

Reply:
Thanks for the hint!